app/services/metrics_service.rb

```diff
@@ -64,6 +66,14 @@ def delivery_attempt_status_changed(status)
       increment("delivery_attempt.status.#{status}")
     end
 
+    def delivery_attempt_pending_status_total(total)
+      OverallMetricsService.gauge("delivery_attempt.pending_status_total", total)
+    end
+
+    def delivery_attempt_total(total)
+      OverallMetricsService.gauge("delivery_attempt.total", total)
+    end
```

Comment:
I wonder if it's better to make these methods of `OverallMetricsService` to make the distinction explicit.